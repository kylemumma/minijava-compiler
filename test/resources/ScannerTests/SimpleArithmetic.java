int a = b + c;
System.out.println(a);