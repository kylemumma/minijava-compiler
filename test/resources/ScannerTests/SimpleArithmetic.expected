INT ID(a) BECOMES ID(b) PLUS ID(c) SEMICOLON PRINT LPAREN ID(a) RPAREN SEMICOLON